### PR TITLE
[READY]Transparent space for MultiZ maps.

### PR DESCRIPTION
**Pull request by Dennok**
##  About The Pull Request

Yes open/space/transparent/openspace
i think, this better then open/transparent/openspace/space

Add ~~bunch of copypasta~~ transparent subtype of space.
Replace outer openspate to space variant on MultiZ debug map.

When you place plating on space all space/openspace turfs above become transformed to openspace with air (with zero moles and pressure, i checked this). 
When you remove plating to space all openspace  turfs above transformed to airless space.

special for #53621

fix space don't call multiz_turf_new()

Replace 2D space by 3D variant on initialization, if it possible.

## Why It's Good For The Game

MultiZ great
Easy map multiZ'ing

## Changelog
:cl:
add: Add transparent subtype of space.
fix: space don't call multiz_turf_new()
add: Replace 2D space to by 3D variant on initialization in multiZ maps.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/space/space.dm [hunk 1]
```diff
@@ -253,3 +253,189 @@
 	destination_x = dest_x
 	destination_y = dest_y
 	destination_z = dest_z
+
+/turf/open/space/transparent
+	baseturfs = /turf/open/space/transparent/openspace
+	intact = FALSE //this means wires go on top
+
+/turf/open/space/transparent/Initialize() // handle plane and layer here so that they don't cover other obs/turfs in Dream Maker
+	. = ..()
```
Comment (Cyberboss) [inline code/game/turfs/open/space/space.dm:262]:
```suggestion
	..()
```
Since you return something else anyway

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by msgerbs** [comment]
What are the implications of this? Does this enable space tiles to be open to the level below them like we have with other tiles?

**Comment by Qustinnus** [comment]
yes

**Comment by Dennok** [comment]
I think it ready for now

**Comment by Dennok** [comment]
I think, it ready for now.

**Comment by AnturK** [comment]
Why this needs separate type instead of internal properties of openspace ?

**Comment by Dennok** [comment]
> Why this needs separate type instead of internal properties of openspace ?

First i try make space from openspace but space has many different fluff, like isspace() check, starlight, etc.
So i make openspace from space, just add visualisation and openspace proc that allow move up/down.
Also i add check to space init to replace it to aired variant if it has aired turf below.

**Comment by Dennok** [comment]
Maybe better change all space to transparent subtype or to turf/open/transparent/openspace/space

**Comment by Dennok** [comment]
Maybe better change type from  `open/space/transparent/openspace` to `open/space/multiz` ?

**Comment by Dennok** [comment]
Maybe open space component? What can go wrong.
Or better, element?

**Pull request closed without merging**